### ISSUE TITLE
sentry example only works on client side

### ISSUE THREAD

**Issue by landvibe** [zeit/next.js#3439]
[The sentry example](https://github.com/zeit/next.js/tree/canary/examples/with-sentry) does not handle server side exception.
I wonder why it is implemented in componentDidCatch which is called only on client side.
How about implementing it in pages/_error.js which can handle both side exceptions.
```js
export default class ErrorPage extends React.Component {
  static getInitialProps({ res, err }) {
    const statusCode = res ? res.statusCode : err ? err.statusCode : null;
    if (IS_CLIENT) {
      // raven-js send event
    } else {
      // raven-node send event
    }
    return { statusCode };
  }
  ...
}
```

**Comment by landvibe** [comment]
closed this because there is similar issue #1852.

**Comment by landvibe** [comment]
closed this because there is similar issue #1852.
